frontend/src/store/SideNavState.ts

```diff
@@ -0,0 +1,20 @@
+import { shallowReactive } from 'vue'
+
+let screenSize = window.innerWidth
+
+window.addEventListener("resize", (event) => {
+    screenSize = window.innerWidth
+    sideNavState.isVisible = sideNavState.isDesktop()
+    sideNavState.isDesktopSize = sideNavState.isDesktop()
+});
+
+export const sideNavState = shallowReactive({
+    isVisible: screenSize >= 1024 ? true : false,
+    isDesktopSize: screenSize >= 1024,
```

Comment:
um.... so I feel isVisible and isDesktopSize default should both depend on the screen size, let's code them in a consistent way. Either put "true : false" or both, or remove it.

Reply:
Yes, I was looking at that and it is strange. Also, one state variable might be enough. And you don't need "true: false" on line 12, just comparing to line 13.

Reply:
sure, one state is indeed enough.